src/test/regress/expected/date_floor.out

```diff
@@ -33,35 +33,55 @@ SELECT date_floor(TIMESTAMP '2001-02-16 20:38:40', INTERVAL '10 minute');
 SELECT date_floor(TIMESTAMP '2001-02-16 20:38:40', INTERVAL '3 hour');
           date_floor          
 ------------------------------
- Fri Feb 16 18:00:00 2001 PST
+ Fri Feb 16 19:00:00 2001 PST
```

Comment:
We should treat this function as an internal function. Maybe I should rename it. It's only purpose is to create fixed size time buckets of duration equal to the interval specified.